modules/web-console/frontend/app/components/page-queries/components/queries-notebook/controller.ts

```diff
@@ -2130,9 +2261,13 @@ export class NotebookCtrl {
 
     async removeParagraph(paragraph: Paragraph) {
```

Comment:
When removing instances of Paragraph, you might want to dispose of Subjects by calling [`Subject.complete`](https://github.com/ReactiveX/rxjs/blob/6.4.0/src/internal/Subject.ts#L91).

Reply:
Added closing of the subjects.